dtfs-central-api/test-helpers/test-data/fee-record-payment-entity-group.ts

```diff
@@ -0,0 +1,24 @@
+import { FeeRecordStatus, Currency, FeeRecordEntityMockBuilder, UtilisationReportEntityMockBuilder } from '@ukef/dtfs2-common';
+import { FeeRecordPaymentEntityGroup } from '../../src/types/fee-record-payment-entity-group';
+
+export const createFeeRecordEntityPaymentGroupForSingleFeeRecord = (
```

Comment:
I'm a bit confused here: 

- File name is `fee-record-payment-entity-group`
- Function name is `...Fee Record Entity Payment Group For Single Fee Record`

As in one says "fee record entity ...", but the other says "fee record payment entity ..."

Reply:
Sorry yes, thanks for spotting, I've updated the naming to match the return type of `FeeRecordPaymentEntityGroup`